.azure-pipelines/matrix.yml

```
@@ -31,10 +40,14 @@ jobs:
           - script: |
               source "${HOME}/conda/etc/profile.d/conda.sh"
               source "${HOME}/conda/etc/profile.d/mamba.sh"
-              echo "  - python="${{ py_vers }} >> environment_test.yml
-              mamba env create -f environment_test.yml
+              cp environment_test.yml environment_test_with_pyversion.yml
+              echo "  - python="$(python.version) >> environment_test_with_pyversion.yml
```

@jcapriot, I'm going back to use the `py_vers` variable for creating the environment so we can keep running these tests on Python 3.8. I'm aware that we should fix that race condition that is happening in Python 3.12, but I think we can do it in #1469. And actually, I prefer using the `py_vers` variable than relying on the default Python version that Azure ships.
```suggestion
              echo "  - python="{{ py_vers }} >> environment_test_with_pyversion.yml
```

---

Yeah, I meant to have it do that in the first place. I had already made that fix in #1469 as well.